--- a/job/basic_data_other_daily_job.py
+++ b/job/basic_data_other_daily_job.py
@@ -4,4 +4,3 @@
 import logging
-import concurrent.futures
 import os.path
 import sys
@@ -74,1 +73,1 @@
-        results = run_check(times)
+        results = run_check(times, date)
@@ -106,19 +105,15 @@
-def run_check(times):
-    data = {}
-    try:
-        with concurrent.futures.ThreadPoolExecutor(max_workers=len(times)) as executor:
-            future_to_data = {executor.submit(stf.fetch_stocks_fund_flow, k): k for k in times}
-            for future in concurrent.futures.as_completed(future_to_data):
-                _time = future_to_data[future]
-                try:
-                    _data_ = future.result()
-                    if _data_ is not None:
-                        data[_time] = _data_
-                except Exception as e:
-                    logging.error(f"basic_data_other_daily_job.run_check处理异常：代码{e}")
-    except Exception as e:
-        logging.error(f"basic_data_other_daily_job.run_check处理异常：{e}")
-    if not data:
-        return None
-    else:
-        return data
+def run_check(times, date):
+    data = {}
+    try:
+        for _time in times:
+            try:
+                _data_ = stf.fetch_stocks_fund_flow(_time, date)
+                if _data_ is not None:
+                    data[_time] = _data_
+            except Exception as e:
+                logging.error(f"basic_data_other_daily_job.run_check处理异常：代码{e}")
+    except Exception as e:
+        logging.error(f"basic_data_other_daily_job.run_check处理异常：{e}")
+    if not data:
+        return None
+    return data
